src/components/Container.tsx

```diff
@@ -26,18 +34,27 @@ export default function Container({ containerModel }: CotaninerProps) {
       }}
     >
       <Tabs value={tabValue} onChange={handleChange}>
-        {layout.children?.map((chart: TODO) => (
+        {children.map((chart: TODO) => (
           <Tab
             id={`container-tab-${chart.refId}`}
-            label={<Typography color={COLORS.TEXT_PRIMARY}>{chart.label}</Typography>}
+            key={chart.refId}
+            sx={{ fontFamily: 'inherit', maxWidth: 200, minWidth: 100, flex: '1 1 0', alignItems: 'flex-start' }}
```

Comment:
Wonder if it's better to separate the styling in a const or css/less file. What do you think?

Reply:
This was actually a big discussion that we had with Egill as using a `const` similarly with makesStyles was rejected. I would also personally prefer to avoid introducing css/less files into a new MUI project.

The conclusion was that we either add a `styled` MUI component or have the styling inline. I can add a `styled` component but I mostly do that for components that will be re-used.

Reply:
Alright, that makes sense. Then lets go with styling inline.